internal/db/db.go

```diff
@@ -28,6 +34,11 @@ func NewSQLDBFromConfig(config Config) (*sql.DB, error) {
 	var err error
 
 	switch config.Dialect {
+	case DialectMariaDB:
```

Comment:
Maybe I'm missing some context: why do we want to include `MariaDB` here if it's not supported yet?

Reply:
Wanted to explicitly have it in the code to say it's not supported vs. having it just be in the documentation. This way, if someone were to set the dialect to `mariadb`, it would error out.